bpf/lib/nodeport.h

```diff
@@ -2377,7 +2394,42 @@ int tail_nodeport_nat_ingress_ipv4(struct __ctx_buff *ctx)
 	__s8 ext_err = 0;
 	int ret;
 
+#if defined(ENABLE_MASQUERADE_IPV4) && defined(ENABLE_IPV4_MASQ_DUAL_PORT_RANGE)
+	/*
+	 * If expanded masquerade port ranges are enabled:
+	 * 1. Initialize the target struct to attempt RevNAT lookup in the new "low band"
+	 * port range (NODEPORT_PORT_NAT_IPV4_RANGE1_MIN/MAX).
+	 */
+	memset(&target, 0, sizeof(target));
+	target.min_port = NODEPORT_PORT_NAT_IPV4_RANGE1_MIN;
+	target.max_port = NODEPORT_PORT_NAT_IPV4_RANGE1_MAX;
 	ret = snat_v4_rev_nat(ctx, &target, &trace, &ext_err);
```

Comment:
Note: `snat_v4_rev_nat` as well as `NODEPORT_PORT_MIN_NAT` are used in `bpf/bpf_overlay.c`.  Does this need to be changed as well to work with dual port ranges?

Also, what about IPv6?